app/controllers/comment.js

```diff
@@ -0,0 +1,131 @@
+const CommentModel = require('../models/Comment')
+
+module.exports = {
+  comment: async (req, res, next) => {
+    const comment = new CommentModel(req.body)
+    try {
+      await comment.save()
+      res.status(201).json({ comment: comment })
+    } catch (error) {
+      console.log(error)
+      res.status(400).json({ error: error })
+    }
+  },
+  delete: async (req, res, next) => {
+    const { id } = req.params
+    console.log(id)
+    try {
+      const comment = await CommentModel.findByIdAndRemove(id)
+      if (!comment) {
+        return res.status(400).json({ error: 'No comment exixts' })
+      }
+      res.status(200).json({ comment: comment })
+    } catch (error) {
+      console.log(error)
+      res.status(400).json({ error: error })
+    }
+  },
+  update: async (req, res, next) => {
```

Comment:
@rak-shit Comment can be updated by only the admins or the person who posted it, so please add check for the person.

Reply:
I think I have added the condition to check if the comment is posted by the user who is updating the comment.

Reply:
@Rupeshiya Sorry, I did not add it. Thanks for reviewing it :)